### PR TITLE
Crash vendor doesn't pull from shipside weapons vendor

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title. 
## Why It's Good For The Game

So you can't get stuff the crash weapons vendor doesn't have thru the loadout vendor.
## Changelog
:cl:
fix: Crash vendor doesn't pull from shipside weapons vendor anymore
/:cl:

### CODE REVIEW

File: code/__DEFINES/factions.dm [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #define FACTION_PIRATE "Pirate"
 #define FACTION_VALHALLA "Valhalla"
 #define FACTION_SPECFORCE "Special Forces"
+#define FACTION_CRASH "Crash"
```
Comment (Lumipharon) [inline code/__DEFINES/factions.dm:22]: I know it works, but this seems like a real misuse of faction lol

Comment (TiviPlus) [reply code/__DEFINES/factions.dm:22]: agree lol

Comment (ivanmixo) [reply code/__DEFINES/factions.dm:22]: It's the cleanest way to get it done + `FACTION_VALHALLA` exists. I could seperate the vendor/loadout defines and these faction defines, though.

**Comment by TiviPlus** [comment]
eh good enough

**Pull request merged**